### ISSUE TITLE
ctdTrim() should be able to set or return flags

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1409]
(f2f result)

**Comment by dankelley** [comment]
This might require several new args (e.g. which variable should be flagged? what is good value? what is bad value), so instead I'll make an arg that forces it to return a boolean vector.

**Comment by dankelley** [comment]
Fixed in "develop" branch commit 2c5297477a1fb2424f9f1a545070549d0aa3ecc6; below is new test code that is part of the building process:
```R
test_that("ctdTrim indices argument", {
          data(ctd)
          a <- ctdTrim(ctd, method="sbe")
          b <- ctdTrim(ctd, method="sbe", indices=TRUE)
          for (name in names(a[["data"]])) {
            ## Must use oceGetData because [["time"]] grabs 'time' from
            ## the metadata, which is a scalar.
            expect_equal(oceGetData(ctd, name)[b], oceGetData(a, name))
          }
})
```
